content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Creating a self-managed, multi-device multisig wallet"
+date: 2022-02-22T16:03:11.470Z
+lastmod: 2022-02-22T16:03:11.470Z
+weight: 6
+draft: false
+# search related keywords
+keywords: ["mac, macos, windows, linux, ios, android"]
+---
+
+{{< notice note >}}
+This tutorial is for someone that would like their transactions to be signed from multiple devices. In this workflow, we will be using a desktop, laptop, and a mobile phone to create a 2-of-3 multisig wallet. You can also use this workflow as a template to easily create another version of a multisig wallet.
+{{< /notice >}}
+
+{{< notice info >}}
+You must be signed in to your account in order to access the messaging system. <br/>
+You will need to import or create a key for each of the devices you wish to use in the wallet. These wallets can exist as foreign or software keys. You only need one device to create the setup. The keys cannot be the same. If you need help creating keys click <u>[here](/getting-started/createsoftwarekey/)</u>
```

Comment:
Remove ~These wallets can exist as foreign or software keys.~ from here

I think we should make a separate note on "foreign key" in the assigning step.

A foreign key is a software key that you created on another device. You can assign a foreign key to a new wallet, but you cannot use it to sign. Signing must be done locally on the device that the software key was created on.

Reply:
perhaps remove
~You only need one device to create the setup.~ too, since technically you do need to involve multiple devices for the setup (at the minimum, to add keys).